easybuild/easyconfigs/a/ASAGI/ASAGI-1.0-foss-2023a.eb

```diff
@@ -0,0 +1,63 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jonney Huang <jonney.huang@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'CMakeMake'
+
+name = 'ASAGI'
+local_commit_id = 'f672507'
+version = '1.0'
+
+homepage = 'https://github.com/TUM-I5/ASAGI'
+description = """ a pArallel Server for Adaptive GeoInformation.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+sources = [{
+    'git_config': {
+        'url': 'https://github.com/TUM-I5/',
+        'repo_name': 'ASAGI',
+        'commit': local_commit_id,
+        'recursive': True,
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+patches = [
+    ('ASAGI-CMakeList.patch', 0),
+    ('ASAGI-level.h.patch', 0),
+]
+checksums = [
+    {'ASAGI-1.0.tar.gz': '703927701a04850a6da9754427dcf429e455ca825673587f5df389980d7452d7'},
```

Comment:
```suggestion
    {'ASAGI-1.0.tar.gz': '6372802e020c727e91ef5bf623012cf31e60448ec5b4cfb3299c9069c1896dab'},
```

Reply:
You missed the `xz` part

Reply:
Yes. I should have read the warning I got from EB:
> WARNING: Reproducible tarballs of Git repos are only possible when using Python 3.9+ to run EasyBuild.
Skipping checksum verification of ASAGI-1.0.tar.xz since Python < 3.9 is used.

Reply:
And yes, I get `e95a15170a3cf2b8d91fa68a4e6a0ccded867ce6ec41a4fa52cf8c3747160638` when I use Python 3.9.